met-web/src/components/landing/FilterDrawer.tsx

```diff
@@ -0,0 +1,171 @@
+// FilterDrawer.js
```

Comment:
Oops... that's an artifact from me splitting the work into 2 files with comments and then copy-pasting 🤦‍♀️ I'll remove it since it's just naming the file; it should be `tsx` anyways